internal/repository/copy_public_test.go

```diff
@@ -432,18 +450,7 @@ func (suite *CopyPublicTestSuite) TestCopyDirNestedReturnsErrorOnCopyDir() {
 		},
 	}
 	createFileSpecs(specs)
-	dstDirHandle, _ := suite.appFs.Open(suite.dstDir)
-	suite.appFs = failfs.New(
-		suite.appFs,
-		map[string]interface{}{
-			"Open": func(path string) (avfs.File, error) {
-				if path == suite.dstDir {
-					return dstDirHandle, errors.New("FailFS!")
-				}
-				return dstDirHandle, nil
-			},
-		},
-	)
+
```

Comment:
I'm not 100% sure what this fixture was supposed to be testing — the test passes the same without it.  I suspect the original need was refactored out a while back.  Yay?